src/features/session_trace/aggregate/index.js

```diff
@@ -195,7 +195,7 @@ export class Aggregate extends AggregateBase {
   }
 
   #onHarvestFinished (result) {
-    if (result.sent && !result.failed && !this.#scheduler.started) { // continue interval harvest only after first call
+    if (result.sent && !result.failed && !this.everSent) { // continue interval harvest only after first call
```

Comment:
```suggest
if (result.sent && !result.failed && !this.#scheduler.started) { // continue interval harvest only after first call
```

Reply:
I need to undo this suggestion because it breaks.  everSent is set to true before sending, making this statement always falsy if we use it here.